dashboard/db/migrate/20170224150102_remove_activity_id_from_gallery_activities.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveActivityIdFromGalleryActivities < ActiveRecord::Migration[5.0]
+  def change
+    remove_index :gallery_activities, name: :index_gallery_activities_on_user_id_and_activity_id
+    remove_column :gallery_activities, :activity_id, :integer
```

Comment:
Note that I changed the migration to remove the index before the column. Though either order is fine for the `up` direction, the column should be reverted before the index in the `down` direction.